src/Sylius/Bundle/ApiBundle/Doctrine/QueryItemExtension/OrderMethodsItemExtension.php

```diff
@@ -86,7 +108,9 @@ private function applyToItemForDeleteMethod(
             return;
         }
 
-        if ($user instanceof AdminUserInterface && in_array('ROLE_API_ACCESS', $user->getRoles(), true)) {
+        if ($user instanceof AdminUserInterface &&
+            in_array('ROLE_API_ACCESS', $user->getRoles(), true) &&
+            $httpRequestMethodType === Request::METHOD_DELETE) {
```

Comment:
This structure violates PSR-12. https://www.php-fig.org/psr/psr-12/#51-if-elseif-else

```suggestion
        if (
            $user instanceof AdminUserInterface &&
            in_array('ROLE_API_ACCESS', $user->getRoles(), true) &&
            $httpRequestMethodType === Request::METHOD_DELETE
        ) {
```
We need to run an easy coding standard over our codebase again. Or maybe we should plug it to some github actions.